packages/design-tokens/src/tokens/functional/components/bento/base.json

```
@@ -0,0 +1,30 @@
+{
+  "brand": {
+    "Bento": {
+      "gap": {
+        "large": {
+          "value": "var(--base-size-32)"
+        },
+        "medium": {
+          "value": "var(--base-size-24)"
+        },
+        "small": {
+          "value": "var(--base-size-16)"
```

Small gaps should use 20px:
```suggestion
          "value": "var(--base-size-20)"
```